server/src/internalClusterTest/java/org/opensearch/snapshots/SearchableSnapshotIT.java

```diff
@@ -88,7 +88,7 @@ protected Settings.Builder randomRepositorySettings() {
     private Settings.Builder chunkedRepositorySettings() {
         final Settings.Builder settings = Settings.builder();
         settings.put("location", randomRepoPath()).put("compress", randomBoolean());
-        settings.put("chunk_size", 2 << 23, ByteSizeUnit.BYTES);
+        settings.put("chunk_size", randomBoolean() ? (2 << 23) : 1000, ByteSizeUnit.BYTES);
```

Comment:
I'd rather make it parameterized and add another test case for it. You can accept chunk size as an input.

Reply:
Done updated this in latest commit.